sklearn/ensemble/tests/test_gradient_boosting_loss_functions.py

```diff
@@ -273,3 +274,24 @@ def test_init_raw_predictions_values():
         for k in range(n_classes):
             p = (y == k).mean()
         assert_almost_equal(raw_predictions[:, k], np.log(p))
+
+
+@pytest.mark.parametrize('seed', range(5))
```

Comment:
Now the important question is...do we parametrize this on weights?

```py
@pytest.mark.parametrize('seed', range(5))
@pytest.mark.parametrize('weighted', [True, False])
def test_lad_equals_quantile_50(seed, weighted):
    ...
    weights = np.linspace(0, 1, n_samples) ** 2 if weighted else None
    ...
```

Reply:
feel free to do it but...

Reply:
But...you feel it is less readable?

Reply:
i'm fine with either